src/app/containers/ATIAnalytics/params/article/buildParams.js

```diff
@@ -0,0 +1,43 @@
+import atiPageViewParams from '../../atiUrl';
+
+import { getPublishedDatetime } from '../../../../lib/analyticsUtils';
+import {
+  getLanguage,
+  getContentId,
+  getPageIdentifier,
+  getPromoHeadline,
+  getThingAttributes,
+} from '../../../../lib/analyticsUtils/article';
+
+export const buildArticleATIParams = (
+  articleData,
+  requestContext,
+  serviceContext,
+) => {
+  const { platform, isUK, statsDestination, previousPath, origin } =
```

Comment:
Can we not just call `const { platform, isUK, statsDestination, previousPath, origin } = useContext(RequestContext);` in here?

Reply:
If we do this we can ditch/combine `src/app/containers/ATIAnalytics/params/article/index.jsx` as its a pass through

Reply:
I was thinking the functions in buildParams.js files need to be reused by functions that are not necessarily react components. E.g. for injecting custom data that cannot be inferred by a component's props such as the attribute of the clicked element.

Currently, `src/app/containers/ATIAnalytics/params/article/index.jsx` reuses the function in `src/app/containers/ATIAnalytics/params/article/buildParams.js`.